package.json

```diff
@@ -1,41 +1,53 @@
 {
```

Comment:
The setup of package.json, as well as npmignore will need to be iterated on later.  This is a scaffolding.